cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py

```diff
@@ -352,6 +386,12 @@ def _add_iam_role(self):
                         effect=iam.Effect.ALLOW,
                         sid="CloudWatchLogsPolicy",
                     ),
+                    iam.PolicyStatement(
+                        actions=["ec2:AssignPrivateIpAddresses", "ec2:DescribeInstances"],
+                        resources=["*"],
```

Comment:
I think this will be handled in the next commit. I am looking at the CR commit by commit

But the resource is way to permissive

Reply:
The resource cannot be limited because we don't know the instance id or network interfaces that the autoscaling group is going to launch